ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -11,15 +11,15 @@
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
 import org.openmrs.uitestframework.test.TestData;
 
 import static org.junit.Assert.assertTrue;
 
-
-public class TransferToWardServiceTest extends ReferenceApplicationTestBase {
+public class TransferToWardServiceTest extends LocationSensitiveApplicationTestBase {
```

Comment:
Refactored the test class to rather extend `LocationSensitiveApplicationTestBase` for the `setUp()` method implementation to be effective. Though the test data can be created when a test class extends `ReferenceApplicationTestBase` but it can not `create the test visit` implementation

`createTestVisit() `{
`new TestData.TestVisit(testPatient.uuid, TestsData.getAVisitType(), getLocationUuid(homePage)).create();`
}
The `createTestVisit()` implementation can only be effective when a test class extends `LocationSensitiveApplicationTestBase`